go.mod

```diff
@@ -39,6 +39,8 @@ require (
 )
 
 replace (
+	github.com/decred/dcrd/dcrjson v1.0.0 => ../dcrd/dcrjson
+	github.com/decred/dcrd/rpcclient v1.0.1 => ../dcrd/rpcclient
```

Comment:
These replaces need to go away in the final version, and to make CI pass it would be better if you replaced it with your branch instead of an unknown path.